streams/sqlite/connection.cpp

```diff
@@ -408,7 +408,7 @@ bool Connection::save_block(const BlockLocation loc, const Span<const uint8_t> b
 
 	sqlite3 *db = _db;
 
-	sqlite3_stmt *update_block_statement;
+	sqlite3_stmt *update_block_statement = nullptr;
```

Comment:
Initialization happens in the switch statement or exits it (via crash), so why would it need to be initialized here too? Doesn't this incur a cost? Times where I had to fix this kind of warnings, I assigned a default value in the `default` case